Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -375,6 +375,11 @@
     - UniformPrinterMachineCircuitboard
     - FloorGreenCircuit
     - FloorBlueCircuit
+    - MicrowaveMachineCircuitboard
+    - ReagentGrinderMachineCircuitboard
+    - ElectricGrillMachineCircuitboard
+    - BoozeDispenserMachineCircuitboard
+    - SodaDispenserMachineCircuitboard
```

Comment:
just a thought, does it even make sense to have these anymore when they are functionally identical to the chem dispenser? Obviously for spawning flavor, sure, but these items being craftable is kinda strange.

Reply:
pro idea use screwdriver to cycle between chem / soda / booze / fridge

Reply:
this can be added later if necessary, for now I think it's fine